.travis.yml

```diff
@@ -16,6 +16,11 @@ dist: bionic
 jobs:
   include:
     - name: "api"
+      services:
+        - docker
+      before_script:
+      - docker run -d -p 27017:27017 --rm mongo
```

Comment:
No issue here; just wanted clarification on why docker is included

Reply:
It's a faster way to set up a MongoDB database